File: core/src/main/java/org/toml/ide/formatter/settings/TomlCodeStyleSettings.java
/*
 * Use of this source code is governed by the MIT license that can be
 * found in the LICENSE file.
 */

package org.toml.ide.formatter.settings;

import consulo.language.codeStyle.CodeStyleSettings;
import consulo.language.codeStyle.CustomCodeStyleSettings;
import jakarta.annotation.Nonnull;

public class TomlCodeStyleSettings extends CustomCodeStyleSettings {
    public TomlCodeStyleSettings(@Nonnull CodeStyleSettings container) {
        super(TomlCodeStyleSettings.class.getSimpleName(), container);
    }
}

File: core/src/main/kotlin/org/toml/ide/formatter/settings/TomlCodeStyleSettings.kt
/*
 * Use of this source code is governed by the MIT license that can be
 * found in the LICENSE file.
 */

package org.toml.ide.formatter.settings

import consulo.language.codeStyle.CodeStyleSettings
import consulo.language.codeStyle.CustomCodeStyleSettings

class TomlCodeStyleSettings(container: CodeStyleSettings) :
    CustomCodeStyleSettings(TomlCodeStyleSettings::class.java.simpleName, container)
